src/adr/articles/0009-integration-ualpa.md

```diff
@@ -0,0 +1,88 @@
+adr_number: "0009"
+---
+
+## Context
+
+Our [Use a LPA service](https://use-lasting-power-of-attorney.service.gov.uk/home) is the latest service built by the OPG.
+
+It is responsible for all services online after Execution of an LPA. We envision that Modernising will not replace this service and instead any new features in this domain will be built upon this service.
+
+We need to ensure any new work done within Modernising pre execution and internal APIs or database work, is still compatible with Use a LPA or the service is brought in line with changes.
+
+## Technical
+
+### Interoperability - How does this enable the exchange of information
+
+The Use a LPA site currently has no API for internal consumers to interact with as there isn't a need.
+
+It's purpose is to pull data as it needs from the Sirius database via an API.
+
+It only stores within its own database, what it needs to run the service.
+
+Anything to do with using or viewing a LPA sits within its domain/responsibility and therefore has no need for a internal API currently for other services to consume.
+
+### Developer Knowledge - How well known is this in our current skill sets
+
+**Overall**: 10/10
+We have a dedicated multidisciplinary team who are responsible for the upkeep of the service.
+
+It is also in its `Live` phase of the [GDS Assessment](https://www.gov.uk/service-manual/agile-delivery/how-the-live-phase-works)
+
+### Support/Open Source - Is it well supported
+
+Use a LPA is Open Source and has a dedicated, multidisciplinary team working on it.
+
+### Scalability
+
+Use a LPA is built and maintained as Infrastructure as Code via Terraform.
+
+It is able to scale up on demand as required. Any future infrastructure added will follow this same principle.
+
+## Ethics
+
+### Mitigate against being tech deterministic
+
+Use a LPA is written in PHP and the Admin service is written in Golang.
+
+Where possible we want to use Golang in place of PHP in the future so any new work that does not tightly couple itself to the main code base should be done in either Python or Golang.
+
+### Ensure you conduct inclusive research
+
+N/A - the use of this technology does not have an effect on marginalised groups directly.
+
+### Think big and imagine what the impact of your work can be
+
+Having the separation of the Modernising creation domain and the Use a LPA domain (post Execution) will mean a clean split of data and functionality that is easier to manage over time.
+
+If we build an internal API layer that Modernising and Use a LPA can integrate with, we can ensure a well defined data schema and rule set for LPA data.
+
+### Interrogate your data decisions
+
+Use a LPA currently follows the rule of only storing the information it absolutely needs to serve the users. It should maintain this rule and if required to share this information with other services, ensure that the use case is ethically correct and only gives the information that is needed.
+
+### Decision
```

Comment:
I'm not sure I understand the purpose of this ADR.

It refers to features (language, tools, team structure) of the UaL service, and where any concerns are raised, they are surely for the UaL service to record a decision on (maintain or provide a consistant service to Modernising or any other dependant product/service for example) rather than modernising.

The decisions are to keep things separate, when currently only one of these things exists, and to facilitate an as yet unknown set of features.

This feels more like documenting the landscape the Modernising is entering, as the decisions recorded can't be acted upon, because the need doesn't yet exist/isn't yet known.

Reply:
Solid points, I am moving this from an ADR to a article instead and changing the context. The aim of this article is to document that the UaLPA service is its own domain and will be built upon with the team owning this. Update out later today to make that point a bit better